Header header

string[] name
float64[] q
float64[] q_des
float64[] qd
float64[] qd_raw
float64[] qd_des
float64[] effort
float64[] effort_raw
float64[] ff
int32[] temper
